ksql-engine/src/main/java/io/confluent/ksql/statement/ConfiguredStatement.java

```diff
@@ -35,7 +35,9 @@
 
   private final PreparedStatement<T> statement;
   @EffectivelyImmutable
-  private final ImmutableMap<String, Object> overrides;
+  private final ImmutableMap<String, Object> clientProperties;
+  @EffectivelyImmutable
+  private final ImmutableMap<String, Object> serverProperties;
```

Comment:
I guess this is what we originally meant by `overrides` which have to be from `KsqlConfig`.. If so, then call this `configOverrides` ... trying to move away from client and server terminology, since it feels a bit broad

Reply:
Renamed to `configOverrides`